### ISSUE TITLE
[Bug] App does not lock when "EditDialog" of an entry was opened

### ISSUE THREAD

**Issue by steffenstollfuss** [helloworld1/FreeOTPPlus#223]
I noticed today that the FreeOTP+ was not locked after I had opened "Edit Details" dialog and locked the Phone.

1. Open FreeOTP+ app secured with "pin".
2. Open the edit dialog of an entry.
3. Turn off the display and wait unitl the pin timeout appered (for me, around 2 minutes)
4. Turn on display and unlock the phone.
5. Now normally the overview dialog of the FreeOTP+ app is locked, but when the "edit dialog" was opened it is not locked and you can see the secret informations. Ups ;)
5.1 If you go back to overview dialog you have to unlock FreeOTP+ as expected.

gr33tz

**Comment by helloworld1** [comment]
Yes, confirmed the issue that the timeout mechanism only works on the token list screen.

**Comment by trikaphundo** [comment]
I have just realized the very same problem, is this going to be fixed?
